### PR TITLE
Google

### PR DESCRIPTION
**Pull request by YauheniyaBasaranovich**

### CODE REVIEW

File: googleCredentials.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "web": {
+    "client_id": "704533824523-fv1b2akrq571rjjk93upt96hqnffn3n3.apps.googleusercontent.com",
+    "project_id": "exadel-axel-1",
+    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+    "token_uri": "https://accounts.google.com/o/oauth2/token",
+    "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+    "client_secret": "qY0i--wf0F4PKAUTj4MTRszD",
```
Comment (aaneitchik) [inline googleCredentials.json]: Again! Configs like this should not be added to Git, it's called "client_secret" for a reason

File: server/services/google.service.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+/**
+ * Creates message body
+ * @param {String} email
+ * @param {{place: {String}, date: {String}}} interview
+ * @returns {String} body
+*/
+function createMessageBody(email, interview) {
+  const date = new Date(interview.date);
+  const emailRows = [];
+  emailRows.push(`To: ${email}`);
```
Comment (aaneitchik) [inline server/services/google.service.js]: Why .push every time and not just create an array with those values?

File: server/services/google.service.js [hunk 3]
```diff
@@ -122,12 +124,12 @@ async function addEvent(email, interview) {
 function createMessageBody(email, interview) {
   const date = new Date(interview.date);
   const emailRows = [];
-  emailRows.push(`To: ${email}`);
-  emailRows.push('Subject: Exadel notifications');
-  emailRows.push('Content-type: text/html;charset=iso-8859-1');
-  emailRows.push('MIME-Version: 1.0\n');
-  emailRows.push(`You have one new assigned interview ${utils.date.getLocalDate(date)} at ${utils.date.getLocalTime(date)}.`);
-  emailRows.push('For more information check your interview list in Exadel-Axel system.');
+  emailRows[0] = `To: ${email}`;
```
Comment (aaneitchik) [inline server/services/google.service.js]: :) I meant it like this:
```sh
const emailRows = [
    'string1',
    'string2'
];
```

**Comment by ala-n** [comment]
Thanks for update!

**Pull request merged**